spec/provider/json_type_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+module PublicIp
+  module Service
+    class JSONService < JsonType
```

Comment:
Just for consistency with all the other tests, would you consider renaming this class to `JSONTypeService`?